src/Resources/app/storefront/src/scss/category-extend.scss

```diff
@@ -0,0 +1,75 @@
+.category-filter-container {
+  min-width: 290px;
+  label {
+    display: inline-block;
+    width: auto;
+    padding: 0px;
+  }
+}
+
+.show-category-list-item {
+  max-height: 500.313rem;
+  min-height: 1.3125rem;
+  height: auto;
+  opacity: 1;
+  transition: all 0.5s;
+  width: 109%;
```

Comment:
Why an exact width of `109%`? Try to find something more sophisticated without relying on exact values.

Reply:
I think, this won't be an exact value but dynamic because it is on percentage, its size will be adjusted automatically as its parent node increases its width, otherwise, should it go for media queries ?